app/src/renderer/vuex/modules/themes.js

```diff
@@ -17,15 +11,22 @@ export default ({ commit }) => {
   }
   const mutations = {
     loadTheme(state) {
-      let theme = localStorage.getItem("appTheme")
+      const theme = localStorage.getItem("appTheme")
       state.active = theme
     },
     setTheme(state, theme) {
       state.active = theme
       localStorage.setItem("appTheme", theme)
     },
     updateTheme(state, theme) {
-      setCssVars(state.options[theme])
+      const newTheme = state.options[theme]
+      const isWin = navigator.platform.toUpperCase().indexOf("WIN") >= 0
+      if (isWin) {
+        document.documentElement.style.setProperty(`--font-weight`, 400)
+      }
+      for (let key in newTheme) {
```

Comment:
why did you remove this refactoring?

Reply:
Coverage reasons, and difficulty of testing that function since (1) I didn’t feel like mocking document was worth it and (2) it’s not being exported. The function was also not being reused anywhere, so putting it inline makes the code easier to follow.

Reply:
Thank your for clarifying